### DISCUSSION TITLE
Double click on the feature request to edit

### DISCUSSION THREAD

**Discussion by owenjuni** [memospot/memospot#81] [Q&A] [answered]
Can you add a double-click to edit card function? The current editing needs to be operated in more, which is too troublesome. Can you add a double-click to edit or put the edit button next to the emoticon?

**Comment by lincolnthalles** [comment, marked as answer]
Interface changes should be done at [Memos](https://github.com/usememos/memos)' upstream project, so they benefit all Memos user base and don't lead to multiple forks.

The double-click to edit feature once existed, but it created issues with mobile clients. However, a direct edit icon or a keyboard shortcut most likely can be implemented via an enhancement request issue.
